pkg/ebpf/tracer_benchmark_test.go

```diff
@@ -0,0 +1,95 @@
+// +build linux_bpf
+
+package ebpf
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"net"
+	"testing"
+	"time"
+)
+
+func BenchmarkTCPLatency(b *testing.B) {
+	RunEBPFBenchmark(b, benchLatencyEchoTCP(64))
+}
+
+func BenchmarkUDPLatency(b *testing.B) {
+	RunEBPFBenchmark(b, benchLatencyEchoUDP(64))
+}
```

Comment:
Is there any reason to have these be benchmarks ? As far as I can tell, the interesting results are what we get out of BPF maps, and not whatever results come out of the go benchmark framework.

Reply:
Go is still doing the isolation and allows us to easily control how many times it runs the loop (adjusting `b.N`) or the length of time we want it to run `-benchtime`.